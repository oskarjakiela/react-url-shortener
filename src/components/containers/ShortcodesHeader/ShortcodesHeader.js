import React from 'react';

import './ShortcodesHeader.css';
import { Heading, Link } from 'components/ui';


const displayName = 'Shorty-ShortcodesHeader';

const ShortcodesHeader = () => (
  <div className={displayName}>
    <Heading className={`${displayName}__heading`}>Previously shortened by you</Heading>
    <Link className={`${displayName}__link`}>Clear history</Link>
  </div>
);

ShortcodesHeader.displayName = displayName;

export default ShortcodesHeader;
