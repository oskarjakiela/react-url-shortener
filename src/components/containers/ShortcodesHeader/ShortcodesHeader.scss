.Shorty-ShortcodesHeader {
  display: flex;
  margin: 0 -.5rem;
  align-items: flex-end;

  > * {
    margin: 0 .5rem;
  }
}

.Shorty-ShortcodesHeader__link {
  padding-bottom: .0625rem;
}
