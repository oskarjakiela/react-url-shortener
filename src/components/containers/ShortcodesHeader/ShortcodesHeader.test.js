import React from 'react';
import renderer from 'react-test-renderer';

import ShortcodesHeader from './ShortcodesHeader';


it('<ShortcodesHeader /> renders correctly', () => {
  const tree = renderer
    .create(<ShortcodesHeader />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
