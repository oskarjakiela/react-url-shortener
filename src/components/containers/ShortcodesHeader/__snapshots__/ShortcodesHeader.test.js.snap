// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShortcodesHeader /> renders correctly 1`] = `
<div
  className="Shorty-ShortcodesHeader"
>
  <h2
    className="Shorty-Heading Shorty-ShortcodesHeader__heading"
  >
    Previously shortened by you
  </h2>
  <a
    className="Shorty-Link Shorty-ShortcodesHeader__link"
  >
    Clear history
  </a>
</div>
`;
